lisp/forge-gitlab.el

```diff
@@ -146,15 +153,15 @@
       :callback (lambda (value _headers _status _req)
                   (funcall cb cb value)))))
 
-(cl-defmethod forge--fetch-issue-posts ((repo forge-gitlab-repository) cur cb)
-  (let-alist (car cur)
+(cl-defmethod forge--fetch-issue-posts ((repo forge-gitlab-repository) issue cb)
+  (let-alist issue
     (forge--glab-get repo
-      (format "/projects/%s/issues/%s/notes" .project_id .iid)
+      (format "/projects/%s/issues/%s/discussions" .project_id .iid)
```

Comment:
Are "notes" and "discussions" the same thing and the latter is just the new name. If so, then please use a separate commit and talk about that.